config/routes.rb

```diff
@@ -6,7 +6,8 @@
  get "/signup", to: "users#new"
  post "/signup", to: "users#create"
  get "/show/:id",to: "users#show"
- resources :users, only: %i(new create show)
+ get "/edit/:id", to: "users#edit"
+ resources :users, only: %i(new create show edit update index destroy)
```

Comment:
Dùng full 7 action rồi thì ko cần only nữa e
```suggestion
 resources :users
```

Reply:
Dạ để em fix hết ạ